test.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+function myFunction() {
+  // make mobile nav list disappear and appear
+  const z = document.getElementById('welcome-top');
+  const x = document.getElementById('menu-item');
+  if (x.style.display === 'flex') {
+    x.style.display = 'none';
+  } else {
+    x.style.display = 'flex';
+  }
+
+  // set the hamburger icon to close when clicked
+  const y = document.getElementById('menu-item');
+  if (y.className === 'menu-items') {
+    y.className += ' responsive';
+  } else {
+    y.className = 'menu-items';
+  }
+  const icon = document.getElementsByClassName('fa-bars')[0];
+  if (icon) {
+    icon.className = icon.className.replace('fa-bars', 'fa-times');
+  } else {
+    document.getElementsByClassName('fa-times')[0] = icon;
+    icon.className = icon.className.replace('fa-times', 'fa-bars');
+  }
+
```

Comment:
- [ ] Kindly stick with Figma design. your design is not matched Figma's design. It should appear on a different screen with white background.
### your design
![his](https://user-images.githubusercontent.com/81727988/167523254-316da6ad-8546-438c-8e3a-1d1d538b27c7.png)
### Figma design

![Figma-design](https://user-images.githubusercontent.com/81727988/167523308-31eb4107-3117-4d07-98d9-9c339898d442.png)

Reply:
Noted thank you. I have changed accordingly

![image](https://user-images.githubusercontent.com/59999191/167692357-ebca28e1-199b-4d66-b027-cbd26ad13bf2.png)